src/finder.ts

```diff
@@ -89,21 +90,41 @@ export function findDeclarationNode(program: Node, identifier: Identifier): Node
   return undefined
 }
 
-function containsNode(nodeOuter: Node, nodeInner: Node) {
+function containsNode(nodeOuter: Node, nodeInner: Node): boolean {
   const outerLoc = nodeOuter.loc
   const innerLoc = nodeInner.loc
+
   return (
-    outerLoc &&
-    innerLoc &&
-    ((outerLoc.start.line === innerLoc.start.line &&
-      outerLoc.end.line === innerLoc.end.line &&
-      outerLoc.start.column <= innerLoc.start.column &&
-      outerLoc.end.column >= innerLoc.end.column) ||
-      (outerLoc.start.line < innerLoc.start.line && outerLoc.end.line >= innerLoc.end.line) ||
-      (outerLoc.start.line <= innerLoc.start.line && outerLoc.end.line > innerLoc.end.line))
```

Comment:
@kush1509 could you look through this? Thanks! It is the only change I made to your file.

Reply:
Looks good!